src/utilcode/clrconfig.cpp

```diff
@@ -297,16 +297,25 @@ BOOL CLRConfig::IsConfigEnabled(const ConfigDWORDInfo & info)
         return FALSE;
 }
 
-
-
 // 
 // Look up a DWORD config value.
 // 
 // Arguments:
-//     * info - see file:../inc/CLRConfig.h for details
-//     
+//     * info - see file:../inc/CLRConfig.h for details.
+//
+//     * useDefaultIfNotSet - if true, fall back to the default value if the value is not set.
+//
+//     * acceptExplicitDefaultFromRegutil - if false, only accept a value returned by REGUTIL if it is
+//           different from the default value. This parameter is useful as a way to preserve existing
+//           behavior.
+//
+//     * result - the result.
+//
+// Return value:
+//     * true for success, false otherwise.
+// 
 // static
-DWORD CLRConfig::GetConfigValue(const ConfigDWORDInfo & info)
+DWORD CLRConfig::GetConfigValue(const ConfigDWORDInfo & info, bool acceptExplicitDefaultFromRegutil, /* [Out] */ bool *isDefault)
 {
```

Comment:
I do not understand why we need `acceptExplicitDefaultFromRegutil`. What will not work without it?